node/coinstacks/solana/api/src/types.ts

```diff
@@ -0,0 +1,20 @@
+import { ParsedTransactionWithMeta } from '@solana/web3.js'
+
+export type Transaction = Omit<ParsedTransactionWithMeta, 'slot'>
```

Comment:
using the typed parsed transaction with meta from solana web3

Reply:
Makes sense